identity-issuance/src/main/java/com/android/identity/issuance/funke/FunkeProofingState.kt

```diff
@@ -53,7 +58,35 @@ class FunkeProofingState(
     @FlowMethod
     fun getEvidenceRequests(env: FlowEnvironment): List<EvidenceRequest> {
         return if (token == null) {
-            listOf(EvidenceRequestGermanEid(tcTokenUrl, listOf()))
+            if (!tosAcknowleged) {
+                val message = env.getInterface(Resources::class)!!
+                    .getStringResource("funke/tos.html")!!
+                listOf(EvidenceRequestMessage(
+                    message = message,
+                    assets = emptyMap(),
+                    acceptButtonText = "Continue",
+                    rejectButtonText = "Cancel"
+                ))
+            } else if (!notificationPermissonRequested) {
+                listOf(EvidenceRequestNotificationPermission(
+                    permissionNotGrantedMessage = """
+                        ## Receive notifications?
+                        
+                        If there are updates to your document the issuer will send an updated document
+                        to your device. If you are interested, we can send a notification to make you aware
+                        of when this happens. This requires granting a permission.
+                        
+                        If you previously denied this permission, attempting to grant it again might not do
+                        anything and you may need to manually go into Settings and manually enable
```

Comment:
"manually" is mentioned twice here. Maybe remove one of them?

Reply:
Done. Also updated the source from which I copied and pasted this.